easybuild/easyconfigs/a/Amber/Amber-18-fosscuda-2018b-AmberTools-18-patchlevel-10-8.eb

```diff
@@ -0,0 +1,89 @@
+    'Amber-%%(version)s-AT-%s_use_external_boost_in_packmol_memembed.patch' % ambertools_ver,
+    'Amber-%(version)s_fix_missing_build_target.patch',
+    'Amber-%(version)s_dont_use_ipo.patch',
+    'Amber-%(version)s_test_cuda.patch',
+]
+checksums = [
+    '2060897c0b11576082d523fb63a51ba701bc7519ff7be3d299d5ec56e8e6e277',  # Amber18.tar.bz2
+    'c630fc3d251fcefe19bb81c8c617e0547f1687b6aef68ea526e4e5fff65bea1c',  # AmberTools18.tar.bz2
+    'd1bad000dc155b77cd20787fbe7da5552dfb1d3e2834407824208b51f9518ad1',  # Amber-18-AT-18_fix_hardcoding.patch
+    # Amber-18-AT-18_fix_intel_mpi_compiler_version_detection.patch
+    'f25d0353b045bc769d3938aa698aea40e9e7bd152a2acca4b7fa1d790cf96297',
+    'c95db1c296c3512b818a1b34a65c2ea4cbce46acf70689919c5f42c3bb0799d3',  # Amber-18-AT-18_fix_mkl_include_path.patch
+    # Amber-18-AT-18_use_fftw_from_mkl_or_external.patch
+    '10e18ddd1218e73cb10531d8426c3568d8bdc255da92e682fffd2ee2a4ad1867',
+    # Amber-18-AT-18_use_FFTW_FFT_instead_of_PUBFFT.patch
+    'ebe85219f22869ec22e531592b785d68c15337171acd479660ab4e3ecd07655f',
+    # Amber-18-AT-18_fix_sander_link_with_external_fftw.patch
+    '6f8eefc905a6817062a65ec1ed757b9d782e4ffc250511e750e6d87110e00c52',
+    # Amber-18-AT-18_fix-cpptraj-dont-use-static.patch
+    'c142011166a26164d65287e7bea2e14d6df62c1984caada48410e62af0d5c3da',
+    '7ace48e6e24aa061737506d08e435e2053ca7074eb78fc47888183aabf71cd44',  # Amber-18-AT-18_cpptraj_use_mkl_fft.patch
+    # Amber-18-AT-18_fix_incorrect_omp_directive_rism.patch
+    '9639dcc8d7b4053950f16d2a1706787bbf5748dead05773a7b53ec47ce8817d2',
+    # Amber-18-AT-18_fix_missing_openmp_at_link.patch
+    '2116f52b5051eb07cfe74fe5bd751e237d1792c248dc0e5a204a19aea25ed337',
+    # Amber-18-AT-18_use_CUSTOMBUILDFLAGS_for_nab.patch
+    '5de8b3e93bd1b36c5d57b64089fc40f14b00c6503fdc7437a005d545c4c16282',
+    # Amber-18-AT-18_fix_missing_do_parallel_in_checkrismunsupported.patch
+    '5cf41b908b730efee760e623465e180d8aa93f0c50b688093bd4258cf508eba7',
+    '1f407b7b5ae2d1f291eebdd2bb7c4a120f96305a89a9028bc0307ca150bb20d6',  # Amber-18-AT-18_fix_rism_fftw_lib.patch
+    '9854a15537353d26d5ebd3d3b711c46d30b50c3dae830e5bf5c9cce489f449e1',  # Amber-18-AT-18_ignore_X11_checks.patch
+    'dcfdf064e774b4a55802b660146f1cdebb1226bdd75713de7a783848fc23526e',  # Amber-18-AT-18_use_easybuild_pythonpath.patch
+    # Amber-18-AT-18_use_external_boost_in_packmol_memembed.patch
+    '9493a3dfddf62d6aa1bbb0a92a39e766a2b76a9ff8a9006e0fcc9ee3e5d94aac',
+    '6b09a6548fa18127245e05c79bdca143adf31db349349cb39bfdb8c4f60772a7',  # Amber-18_fix_missing_build_target.patch
+    'a4c12ad39088ce6e9e7bad39e8d9a736989e6ae4a380c70535a5451eb3060903',  # Amber-18_dont_use_ipo.patch
+    '672c7667c899c91d002a4b095f1ac1691f77e7b4095414084f1d4cdc7c860d79',  # Amber-18_test_cuda.patch
+]
+
+dependencies = [
+    ('netCDF', '4.6.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('Python', '2.7.15'),
+    ('Boost', '1.68.0', '-Python-%(pyver)s'),
```

Comment:
@akesandgren Please use `Boost.Python` as a dependency instead, if you need a `Boost` with Python bindings

Reply:
Didn't know it existed :-) Going blind on all eyes apparently.

Reply:
It's fairly recent, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6763

Reply:
Then i should also drop down to 1.67 since that's the only existing Boost for a 2018b toolchain at the moment or?

Reply:
@akesandgren Indeed, for the `2018b` toolchain generation, we're sticking to Boost 1.67.0, to ensure all easyconfigs using a `2018b` toolchain that depend on `Boost` are compatible with each other...

Reply:
Changes done.